packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.test.js

```diff
@@ -292,12 +307,12 @@ describe('getRoutes', () => {
  */
```

Comment:
The description mentions the usage of `setProps` enzyme API, but we no longer use that API.

Reply:
ah thanks for catching that, in enzyme, setProps is used to trigger prop change on a test component, in React testing library, we can do the same trick by using `rerender` method returned from the first time we render the component